packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -1311,49 +1399,54 @@ export class DataTable {
-                            tabIndex={0}
-                            size={buttonSize}
-                            color='secondary'
-                            onKeyUp={(event) =>
-                              (event.code === 'Space' ||
-                                event.code === 'Enter') &&
-                              this.performRowAction(action, row)
-                            }
-                            onClick={() => this.performRowAction(action, row)}
-                            aria-label={action.name}
-                          >
-                            {action.iconName ? (
-                              <fw-icon
-                                name={action.iconName}
-                                library={
-                                  action.iconLibrary
-                                    ? action.iconLibrary
-                                    : 'crayons'
-                                }
-                                size={10}
-                              ></fw-icon>
-                            ) : (
-                              action.name
-                            )}
-                          </fw-button>
-                        </fw-tooltip>
-                      );
-                    }
-                    return actionTemplate;
-                  })}
+                  {this.showRowActionsAsMenu
+                    ? this.renderKebabMenu(row)
+                    : this.rowActions.map(
+                        (action: DataTableAction | DataTableMenuAction) => {
+                          let actionTemplate = null;
+                          if (
+                            !action.hideForRowIds ||
+                            (action.hideForRowIds &&
+                              !action.hideForRowIds.includes(row.id))
+                          ) {
+                            const buttonSize: 'icon-small' | 'small' =
+                              action.iconName ? 'icon-small' : 'small';
+                            actionTemplate = (
+                              <fw-tooltip content={action.name} distance='5'>
+                                <fw-button
+                                  tabIndex={0}
+                                  size={buttonSize}
+                                  color='secondary'
+                                  onKeyUp={(event) =>
+                                    (event.code === 'Space' ||
+                                      event.code === 'Enter') &&
+                                    this.performRowAction(action, row)
+                                  }
+                                  onClick={() =>
+                                    this.performRowAction(action, row)
+                                  }
+                                  aria-label={action.name}
+                                >
+                                  {action.iconName ? (
+                                    <fw-icon
```

Comment:
can we implement the usage of graphicProps here as well ?. Some of the row actions buttons might also have a usecase to have icon with color.

Reply:
Yes we can. So we need to support graphicsProps as well as iconName and iconLibrary which were previously used right?